tagreader/web_handlers.py

```diff
@@ -38,12 +41,26 @@ def get_url_pi():
 
 
 def get_auth_aspen():
-    return HTTPKerberosAuth(mutual_authentication=OPTIONAL)
+    if internal:
+        return HTTPKerberosAuth(mutual_authentication=OPTIONAL)
+    else:
+        from .BearerAuth import BearerAuth
+
+        tenantID = '3aa4a235-b6e2-48d5-9195-7fcf05b459b0'
+        clientID = '7adaaa99-897f-428c-8a5f-4053db565b32'
+        scopes = [
+            "https://ewepwapa1pep04-statoilsrm.msappproxy.net/ProcessExplorer/ProcessData//user_impersonation"]
+        auth = BearerAuth.get_bearer_token(
+            tenantID=tenantID, clientID=clientID, scopes=scopes, verbose=True)
+        return auth
 
 
 def get_url_aspen():
-    # internal url (redirects to dll)
-    return r"https://aspenone.api.equinor.com"
+    if internal:
+        # internal url (redirects to url including AtProcessDataREST.dll)
+        return r"https://aspenone.api.equinor.com"
+    else:
```

Comment:
Suggest to remove the else statement - the code below will be executed anyway.

Reply:
See above